ci/scripts/clone-build_ci.sh

```diff
@@ -1,11 +1,6 @@
 #!/bin/bash
-set -eux
+set -ux
```

Comment:
I think we want all errors to be trapped. No?
```suggestion
set -eux
```

Reply:
If you fail immediately on error, you can't check for the error and report it to the log (without adding a real bash trap or a script above this one). I think turning off `-e` is the correct move here, just need to be extra careful about checking and handling errors.

Reply:
good point.  I would think we would disable that (set +e) where we expect to fail.

Reply:
@WalterKolczynski-NOAA Correct!

Reply:
I think Rahul is right, let's just turn off `-e` where we will encounter errors with the PR, like checkout and build (basically where there are already status checks).

Reply:
OK, let me look at that and fine tune it.  I just caught Rahul's `set +e `suggesting, sounds like a plan.